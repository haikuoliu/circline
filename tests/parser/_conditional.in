aList = ["str1","str2","str3"];
for(int i =0; i<= 5; i++)
    if (str == "str2"){
        break;
    }
    else{
        /* do something */
    }
}
