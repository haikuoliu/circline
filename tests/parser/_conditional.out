Expr(Assign(aList, List(String_Lit(str1), String_Lit(str2), String_Lit(str3))));
For(Local(int, i, Num_Lit(0));Binop(Id(i), Leq, Num_Lit(5));Assign(i, Binop(Id(i), Add, Num_Lit(1)))){If(Binop(Id(str), Equal, String_Lit(str2))){Expr(Binop(Num_Lit(3), Add, Num_Lit(2)));} Else{}}
